blog-post.html

```diff
@@ -1,12 +1,138 @@
+        park four dollar toast mumblecore vaporware.
+
+        Taiyaki vaporware franzen swag. Migas single-origin coffee intelligentsia pork belly. Fanny pack affogato echo
+        park meh hella roof party meggings, poutine microdosing schlitz cred unicorn try-hard. 8-bit brooklyn
+        farm-to-table heirloom blue bottle tilde pug la croix knausgaard chambray DIY tofu church-key.
+
+        Pitchfork pickled disrupt bitters meh lo-fi.</br></br>
+        &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Microdosing
+        bicycle rights flexitarian +1 glossier
+        selvage. Mlkshk vegan fanny pack freegan, migas asymmetrical waistcoat mustache godard chia activated charcoal
+        sustainable. Flexitarian listicle distillery crucifix coloring book bushwick palo santo. 3 wolf moon church-key
+        snackwave fashion axe before they sold out cred raw denim gluten-free artisan. Banjo sartorial prism photo booth
+        freegan enamel pin, shabby chic church-key. Bespoke pour-over mumblecore, flannel try-hard DIY chambray hella
+        tote bag intelligentsia lomo coloring book.
+
+        Air plant umami palo santo ennui deep v echo park wolf. Kickstarter 3 wolf moon readymade activated charcoal.
+        Irony leggings pitchfork vice post-ironic vaporware. Prism lumbersexual iPhone helvetica. Authentic shabby chic
+        actually, pinterest live-edge blue bottle bitters tilde direct trade godard. Pinterest vice artisan, sriracha
+        letterpress put a bird on it echo park PBR&B copper mug cold-pressed air plant kickstarter.
+      </h2>
+
+      <div>
+      </div>
+    </main>
+    <!--Footer section start-->
+    <section class="footer">
+      <span class="blogcomment"> Questions or Comments?</span> &nbsp &nbsp&nbsp<span id="bold">GET IN TOUCH:</span>
+      <div class="footercontent">
+        <p20>Lorem ipsum dolor sit amet consectetur adipisicing elit. Praesentium consequuntur iusto obcaecati aut
+          placeat ipsam quo, explicabo earum soluta illo. Lorem, ipsum dolor sit amet consectetur adipisicing elit. Et,
+          accusamus!</p20>
+          <!--Media Labels and print-->
+        <img class='media' img src="./footerlink-dribbble.png">
+        <p id='drib'>dribbbl.com</p>
+        <img class='media1' img src="footerlink-email.png">
+        <p id='mail'>82@HOTMAIL.com</p>
+        <img class='media2' img src="footerlink-facebook.png">
+        <p id='fb'>facebook.com</p>
+        <img class='media3' img src="footerlink-home.png">
+        <p id='address'> 69 the Road to Nowhere, Westchester</p>
+        <img class='media4' img src="footerlink-instagram.png">
+        <p id="IG">instagram.com</p>
+        <img class='media5' img src="footerlink-phone.png">
+        <p id="num">(069)420-0420</p>
+        <img class="media6" img src="footerlink-twitter.png">
+        <p id="twitt">@bigkahunas</p>

```

Comment:
All of these images have almost identical css on them (apart from your `right: (x)px` and your `top: (x)px` and there are better ways to do that positioning). Since they are all almost identical you could have targeted each one with the same CSS property to cut back on how many you needed to write out